manifests/params.pp

```diff
@@ -229,7 +229,7 @@
       $config        = '/etc/inet/ntp.conf'
       $driftfile     = '/var/ntp/ntp.drift'
       $keys_file     = '/etc/inet/ntp.keys'
-      if $::operatingsystemrelease =~ /^(5\.10|10|10_u\d+)$/
+      if $::kernelrelease =~ /^(5\.10|10|10_u\d+)$/
```

Comment:
By changing this to use the `kernelrelease` fact, I think we can clean up the conditional to only check for `5.10`.